<launch>

<param name="use_sim_time" type="bool" value="false"/>

<node pkg="VISFS" type="VISFSInterfaceROSNode" name="VISFSInterfaceROSNode" output="screen">
    <remap from="left/image"            to="/camera/infra1/image_rect_raw"/> 
    <remap from="right/image"           to="/camera/infra2/image_rect_raw"/>
	<remap from="left/info"				to="/camera/infra1/camera_info"/>
    <remap from="right/info"           	to="/camera/infra2/camera_info"/>
    <param name="left/image_transport" 	type="string"       value="compressed"/>
	<param name="right/image_transport" type="string"       value="compressed"/>

    <param name="approx_sync"           type="bool"			value="true"/>
	<param name="queue_size"			type="int"			value="10"/>
    <param name="camera_frame_id"       type="string"       value="camera_link"/>
    <param name="robot_frame_id"        type="string"       value="base_link"/>
    <param name="publish_tf"            type="bool"         value="false"/>

    <param name="base_line"             type="double"       value="0.05"/>

    <param name="System/Sensor"         type="int"          value="0"/>
    <param name="System/Monitor"        type="bool"         value="true"/>
    <param name="Tracker/MaxFeatures"   type="int"          value="200"/>
    <param name="Tracker/FlowBack"      type="bool"         value="false"/>
    <param name="Estimator/PnPFlags"    type="int"          value="1"/>
    <param name="Estimator/PnPReprojError"  type="double"   value="2"/>
</node>

</launch>